store/dataset.js

```diff
@@ -210,7 +243,7 @@ export const actions = {
     commit('setFilter', newState.filter)
   },
   async loadData({ state, dispatch }) {
-    await dispatch('discovery/getDatasetsAndPopulateFileLists', null, {
+    await dispatch('nid/getDatasetsAndPopulateFileLists', null, {
```

Comment:
Actually, this calls adds the "nid" bit on, so maybe it shouldn't be in the settings!

Reply:
Thats the name of the store file to use it isn't to do with the url thats called